src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceAboutToSubmitHandler.java

```diff
@@ -139,7 +140,17 @@ public static void checkWarningsAndErrors(SscsCaseData sscsCaseData, ScannedDocu
             }
         }
 
-        if (ScannedDocumentType.URGENT_HEARING_REQUEST.getValue().equals((scannedDocument.getValue().getType()))
+        String scannedDocumentType = scannedDocument.getValue().getType();
+        String actionCode = sscsCaseData.getFurtherEvidenceAction().getValue().getCode();
+
+        if (ScannedDocumentType.SET_ASIDE_APPLICATION.getValue().equals(scannedDocumentType)
```

Comment:
Seems like changes in this file and the test file were included unintentionally in this adjournment PR.